_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -59268,8 +59268,7 @@
 	},
 /obj/machinery/atmospherics/unary/vent_pump/high_volume{
 	dir = 4;
-	frequency = 1379;
-	id_tag = "robotics_solar_pump"
+	autolink_id = "robotics_solar_pump"
```

Comment:
Yes. Aft Port solars are still called robotics solars on these IDs.

I am willing to bet most of you here have no idea why they would even be called that given they are on the opposite side of the station, and to be fair, robotics was a part of engineering in like.. 2012?